Pop Alexia/matrice 2/main.c

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{
+    int matrix[4][4],i,j;
```

Comment:
Maybe read the size of the matrix from the keyboard and allocate it dynamically - get used to that, you'll need it!